psalm.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="1"
+    findUnusedPsalmSuppress="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <!-- findUnusedCode="true" -->
```

Comment:
Can't be enabled until example domain is in use. Possibly not usable at all, since I will need to leave intentionally unfinished code in the repository, for workshop attendees to complete